scripts/us_eia/opendata/process/coal.py

```diff
@@ -39,38 +39,40 @@ def extract_place_statvar(series_id, counters):
         sv_id = f'COAL.{measure}.{code}.{period}'
         return (place, sv_id, True)
 
+    # TODO: Ignore this pattern until we have a way to model multi-location SV's.
     # Pattern #2: COAL.{EXPORT|IMPORT}_{Measure}.{Type}-{CountryIso}-{UsPortIso}.{Period}
     # Pattern #3: COAL.{SHIPMENT}_{Submeasure}.{Source}-{Destination}-{Material}.{Period}
-    m = re.match(r"^COAL\.([A-Z]+)_([A-Z]+)\.([^-]+)-([^-]+)-([^.]+)\.([AQM])$",
-                 series_id)
-    if m:
-        activity = m.group(1)
-        measure = m.group(2)
-        if activity in ['EXPORT', 'IMPORT']:
-            # Pattern #2
-            # TODO: model destination / source port as well
-            type = m.group(3)
-            place = m.group(4)
-            port = m.group(5)
-            period = m.group(6)
-            sv_id = f'COAL.{activity}_{measure}.{type}.{period}'
-            return (place, sv_id, False)
-        elif activity == 'SHIPMENT':
-            # Pattern #3
-            source = m.group(3)
-            if source.isalpha():  # could include 3-letter region codes
-                destination_power_plant = m.group(4)
-                material = m.group(5)
-                period = m.group(6)
-                sv_id = f'COAL.SHIPMENT_{measure}.{material}.{period}'
-                return (source, sv_id, True)
-            else:
-                # TODO: Handle remaining places - coal mines
-                counters[f'error_unknown_coal_mine SHIPMENT '] += 1
-                return (None, None, None)
-        else:
-            counters[f'unknown #2,3 activity ({activity})'] += 1
-            return (None, None, None)
+    # m = re.match(r"^COAL\.([A-Z]+)_([A-Z]+)\.([^-]+)-([^-]+)-([^.]+)\.([AQM])$",
```

Comment:
nit: maybe delete this with a link to https://github.com/beets/data/blob/c6166b28abdfa5bcabc00b4544c1f10e5731ba96/scripts/us_eia/opendata/process/coal.py#L44-L73 for future reference

Reply:
I'd like to keep that, if it's ok? It took a while to figure out, and I sometimes go on cleaning sprees of idle branches so don't want to lose it. I hope we can uncomment them out soon..